### DISCUSSION TITLE
All Users receiving intercom messages intended only for back end

### DISCUSSION THREAD

**Discussion by Disasterpetrescue** [ushahidi/platform#4238] [General]
Hello. All users assigned no User permissions other than having their email set up, is receiving messages that are intended only for back end users asking if they would like to use the USSD integration. 

https://disasterpetrescue.ushahidi.io/views/map

**Comment by tuxpiper** [comment]
Thank you for this observation. We have since adjusted the Intercom rules so that this message is not appearing to general user groups.
